src/utils/nodes/BinaryExpression.ts

```diff
@@ -0,0 +1,31 @@
+import { NodeClass, NodeClassConstructor } from './Node.abstract';
+
+export class BinaryExpressionClass extends NodeClass {
+	constructor(params: NodeClassConstructor) {
+		super(params);
+	}
+
+	serialize = () => {
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any
+		const binaryExpression = this.node as any;
+		const left = binaryExpression.left.value;
+		const right = binaryExpression.right.value;
+		const operator = binaryExpression.operator;
+		switch (operator) {
+			case '+':
+				return left + right;
+			case '-':
+				return left - right;
+			case '/':
+				return left / right;
+			case '*':
+				return left * right;
+			case '%':
+				return left % right;
+			default:
+				return 'operator not supported';
+		}
+	};
+
+	traverse = () => {};
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously. -->
🎯 suggestion(blocking):
I suggest to traverse binaryExpression members further.
Otherwise, they won't be properly managed by the event loop.
Example:
`console.log(1 + console.log(1));`